go/sns/PublishMessage/PublishMessage.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+// snippet-start:[sns.go.publish_message]
+package main
+
+// snippet-start:[sns.go.publish_message.imports]
+import (
+	"flag"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/aws/aws-sdk-go/service/sns/snsiface"
+)
+// snippet-end:[sns.go.publish_message.imports]
+
+// PublishMessage publishes a message to an Amazon SNS topic
+// Inputs:
+//     svc is an Amazon SNS service object
+//     msg is the message to publish
+//     topicARN is the ARN of the topic to publish through
```

Comment:
... the **Amazon Resource Name (ARN)**...

Reply:
Fixed